src/chain.h

```diff
@@ -459,6 +459,9 @@ class CChain {
     /** Set/initialize a chain with a given tip. */
     void SetTip(CBlockIndex *pindex);
 
+    /** Ensure all members of this chain have a truthy nTx/nChainTx value. */
```

Comment:
In commit "chain: add unused CChain::FakeNTx" (ebc844e8a1bacfe603db85ef4deae6e1f109cd57)

Note: I don't know if this comment is true, but it _feels_ true.

Reply:
re: https://github.com/bitcoin/bitcoin/pull/15606#discussion_r271916764

Now I'm confused about this comment. I thought it meant truthy in the Colbert sense (https://en.wikipedia.org/wiki/Truthiness). But that doesn't work where truthy is used later in https://github.com/bitcoin/bitcoin/commit/8d9b707f609b647fb28ae182a097adc03e718725#diff-349fbb003d5ae550a2e8fa658e475880R777. Since this is C++, it would probably be clearer to say "non-zero" here and "non-null" there, if that's what is meant.

Reply:
Ah sorry, I guess "truthy" is colloquial to Python: https://stackoverflow.com/questions/39983695/what-is-truthy-and-falsy-in-python-how-is-it-different-from-true-and-false